libs/payments/paypal/src/lib/paypalBillingAgreement.manager.ts

```diff
@@ -4,59 +4,61 @@
 
 import { Injectable } from '@nestjs/common';
 
-import { SubscriptionManager } from '@fxa/payments/stripe';
 import { PayPalClient } from './paypal.client';
 import { BillingAgreement, BillingAgreementStatus } from './paypal.types';
 import { PaypalCustomerMultipleRecordsError } from './paypalCustomer/paypalCustomer.error';
 import { PaypalCustomerManager } from './paypalCustomer/paypalCustomer.manager';
-import { PaypalManagerError } from './paypal.error';
+import { PaypalBillingAgreementManagerError } from './paypal.error';
 
 @Injectable()
-export class PayPalManager {
+export class PaypalBillingAgreementManager {
   constructor(
     private client: PayPalClient,
-    private subscriptionManager: SubscriptionManager,
     private paypalCustomerManager: PaypalCustomerManager
   ) {}
 
-  public async getOrCreateBillingAgreementId(
+  public async getOrCreatePaypalBillingAgreementId(
```

Comment:
Since this is now the `PaypalBillingAgreementManager`, it's no longer necessary for `PaypalBillingAgreement` to be part of the method names.